network/lldpd/lldpd.yaml

```diff
@@ -0,0 +1,54 @@
+#© 2024 Nokia
+#Licensed under the Mozilla Public License 2.0
+#SPDX-License-Identifier: MPL-2.0
```

Comment:
```suggestion
# © 2024 Nokia
# Licensed under the Mozilla Public License 2.0
# SPDX-License-Identifier: MPL-2.0
```

Reply:
Unsure whether this is a must, but with spaces it seems more consistent with overall codestyle